docs/path-resolution.rst

```diff
@@ -0,0 +1,470 @@
+.. _path-resolution:
+
+***************
+Path Resolution
```

Comment:
Maybe "Import Path Resolution"?

Reply:
Well, it's not just about imports (also for CLI arguments and paths used in JSON) so I think "Path Resolution" is more accurate but I can change it if you think it gives people reading the docs a better idea of that this section is about.

Reply:
ok!

Reply:
Changed the header. I left the file name and reference as is.